src/data/auroradb.ts

```diff
@@ -691,3 +692,39 @@ export const getPassengerTypeByNameAndNocCode = async (
         throw new Error(`Could not retrieve passenger type by nocCode from AuroraDB: ${error}`);
     }
 };
+
+type SavedPassengerType = { group: GroupPassengerType[]; single: PassengerType[] };
```

Comment:
We usually go for interfaces instead of types - do we need to have this be a type?

Reply:
Have updated, might look at a linting rule for this in future if this is the way we want to go